contracts/NomoPlayersDropMechanic.sol

```diff
@@ -190,6 +198,62 @@ contract NomoPlayersDropMechanic is Ownable, ReentrancyGuard {
         emit LogWhitelistedSet(beneficiaries);
     }
 
+    /**
+     * @notice Sets privileged members who can claim free token.
+     * @param members address[] representing the members will be privileged
+     */
+    function setPrivileged(address[] memory members) public onlyOwner {
+        require(
+            members.length > 0 && members.length <= 100,
+            "Privileged members array length must be in the bounds of 1 and 100"
+        );
+
+        for (uint256 i = 0; i < members.length; i++) {
+            privileged.push(members[i]);
+        }
+
+        emit LogPrivilegedSet(members);
+    }
+
+    /**
+     * @notice Transfers ERC721 token to `n` number of privileged users.
+     
+     * @dev Deployer executes airdrop.
+     * NomoPlayersDropMechanic distributes one token to each of the privileged addresses if the requirements are met.
+     *
+     * Requirements:
+     * - the caller must be owner.
+     */
+    function executeAirdrop() public onlyOwner {
+        require(!isAirdropExecuted, "Airdrop has been executed");
+        require(
+            (tokens.length >= privileged.length) && (privileged.length > 0),
+            "Invalid airdrop parameters"
+        );
+        uint256[] memory airdroppedTokens = new uint256[](privileged.length);
+        claimedTokens[msg.sender] += privileged.length;
```

Comment:
claimedTokens should be located in the for loop and should be assigned to the `privileged` account, rather than the `msg.sender`